.github/workflows/changesets.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
```

Comment:
Should we also constrain this GitHub action so that it only runs when changes are made in one of our packages?

For example:

```
on:
  push:
    branches:
      - main
    paths:
      - 'packages/**'
```